submit_cycle.sh

```diff
@@ -39,21 +39,24 @@ export do_DA=YES  # do full DA update
 do_hofx=NO  # use JEDI to calculate hofx, but do not do update 
              # only used if do_DA=NO  
 export ASSIM_IMS=NO
-export ASSIM_GHCN=YES
+export ASSIM_GHCN=NO
 export ASSIM_SYNTH=NO
+export ASSIM_GTS=YES
+export CYCHR=6
 
 DAtype="letkfoi_snow" # options: "letkfoi_snow" , "letkf_snow"
 
 ############################
 # set your directories
 
 CYCLEDIR=$(pwd)  # this directory
-export WORKDIR=/scratch2/BMC/gsienkf/Clara.Draper/workdir/ # temporary work dir 
-export OUTDIR=${CYCLEDIR}/exp_out/${exp_name}/output/      # directory where output will be saved
-ICSDIR="/scratch2/BMC/gsienkf/Clara.Draper/DA_test_cases/offline_ICS/single/" # OUTDIR for experiment with initial conditions
+export WORKDIR=/scratch1/NCEPDEV/global/Jiarui.Dong/JEDI/workflow/experiment/workdir # temporary work dir 
+export OUTDIR=/scratch1/NCEPDEV/global/Jiarui.Dong/JEDI/workflow/experiment/output      # directory where output will be saved
+export ICSDIR=$OUTDIR
+#ICSDIR="/scratch2/BMC/gsienkf/Clara.Draper/DA_test_cases/offline_ICS/single/" # OUTDIR for experiment with initial conditions
                                                            # will use ensemble of restarts if present, otherwise will try 
                                                            # to copy a non-ensemble restart into each ensemble restart
-                                
+
```

Comment:
Please change all variables above this line back to my values.

Reply:
And set CYCHR to 24

Reply:
Done